bin/plugin/commands/release.js

```diff
@@ -650,17 +487,20 @@ async function releaseRC() {
 			' Team.\n'
 	);
 
-	const release = await releasePlugin( true );
+	const version = await releasePlugin( true );
 
 	log(
 		'\n>> 🎉 The ' +
 			config.name +
 			' version ' +
-			formats.success( release.name ) +
-			' has been successfully released.\n',
-		'You can access the GitHub release here: ' +
-			formats.success( release.html_url ) +
+			formats.success( version ) +
+			' has been successfully tagged.\n',
+		"In a few minutes, you'll be able to find the GitHub release draft here: " +
+			formats.success(
+				`https://github.com/${ config.githubRepositoryOwner }/${ config.githubRepositoryName }/releases/`
+			) +
 			'\n',
+		"Don't forget to publish the release once the draft is available!\n",
```

Comment:
Should update the wording it's an actual published release right?

Reply:
Yes! Good spot, thanks!

Reply:
Fixed in 6bb426b.